dccrg.hpp

```diff
@@ -3279,22 +3279,24 @@ template <
 				No need to update local neighbors_to of refined cell, if they are larger
 				they will also be refined and updated.
 				*/
```

Comment:
This comment is maybe up for debate, but this fix allows me to run Flowthrough_amr on 4 tasks with 2 levels on an 8x4x4 grid.